content/quickstart/java/metrics.md

```diff
@@ -127,15 +141,14 @@ import java.io.InputStreamReader;
 
 public class Repl {
     public static void main(String ...args) {
+        // Step 1. Our OpenCensus initialization will eventually go here
```

Comment:
This is unnecessary since `Step 1` is non-existent but also detracts from the plain code sample.